docs/src/whatsnew/latest.rst

```diff
@@ -72,11 +72,11 @@ This document explains the changes made to Iris for this release
 💼 Internal
 ===========
 
-#. `@trexfeathers`_ used the `Pull Request Labeler Github action`_ to add the
-   ``benchmark_this`` label (:ref:`more info <on_demand_pr_benchmark>`) to
-   pull requests that modify ``requirements/locks/*.lock`` files - ensuring
-   that we know whether dependency changes will affect performance.
-   (:pull:`5763`)
+#. `@trexfeathers`_ setup automatic benchmarking on pull requests that modify
+   ``requirements/locks/*.lock`` files - ensuring that we know whether
+   dependency changes will affect performance. Such pull requests are also
+   labelled using the `Pull Request Labeler Github action`_ to increase
+   visibility. (:pull:`5763`, :pull:`5776`)
```

Comment:
This needs to be updated to reflect the changes since the initial commit when it was added.